lib/game.rb

```diff
@@ -0,0 +1,61 @@
+# Understands how to move through the big steps of tic-tac-toe.
+
+class Game
+  def initialize(player = Player, messages = Messages.new, board = Board.new, rules = Rules.new, parser = Parser.new)
+    @board = board
+    @parser = parser
+    @rules = rules
+    @messages = messages
+    @player_one = player.create(:X)
```

Comment:
Separation of responsibilities: Should the game be in charge of the player symbols?